charts/presto/templates/hive-server-statefulset.yaml

```diff
@@ -97,3 +101,8 @@ spec:
         emptyDir: {}
       - name: hive-metastore-db-data
         emptyDir: {}
+{{- if and .Values.spec.config.sharedVolume.enabled .Values.spec.config.sharedVolume.createPVC: }}
```

Comment:
You don't need an and condition here, nor `.Values.spec.config.sharedVolume.createPVC`